ci/jenkins/test.sh

```diff
@@ -419,6 +419,29 @@ function deliver_antrea {
     done
 }
 
+function generate_ssh_config {
+    echo "=== Generate ssh-config ==="
+    SSH_CONFIG_DST="${WORKDIR}/.ssh/config"
+    echo -n "" > "${SSH_CONFIG_DST}"
+    kubectl get nodes -o wide --no-headers=true | awk '{print $1}' | while read sshconfig_nodename; do
+        echo "Generating ssh-config for Node ${sshconfig_nodename}"
+        sshconfig_nodeip="$(kubectl get node "${sshconfig_nodename}" -o jsonpath='{.status.addresses[0].address}')"
+        if [[ ! "${sshconfig_nodeip}" =~ ^[0-9]+(\.[0-9]+){3}$ ]];then
+            sshconfig_nodeip="[${sshconfig_nodeip}]"
```

Comment:
Is it for IPv6?

Reply:
I believe so, but please add a comment @xliuxu

Reply:
Yes. Comment added.

Reply:
@lzhecheng Sure. I think we can skip the tests as the latest change does not hit any code/script.